test/integration/model/json.test.js

```diff
@@ -227,7 +227,7 @@ describe(Support.getTestDialectTeaser('Model'), function () {
           return this.Event.findAll({
             where: {
               data: {
-                employment: null
+                employment: current.dialect.name === 'mysql' ? 'null' : null
```

Comment:
Yeah, it seems wrong to me too, but unfortunately, MySQL returns null as a string, rather than the null value.